packages/design/src/FormManager/common/Notifications/NotificationAlert.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { Meta, StoryObj } from '@storybook/react';
+
+import { NotificationAlert } from './NotificationAlert';
+
+export default {
+  title: 'FormManager/Notifications/NotificationAlert',
+  component: NotificationAlert,
+} satisfies Meta<typeof NotificationAlert>;
+
+export const Info: StoryObj<typeof NotificationAlert> = {
+  args: {
+    id: '1',
```

Comment:
Very minor here. `args.id` is always 1. There could be an opportunity to either have an ID const that gets reused or to do a default object that is spread in the stories here.

Reply:
Thanks, that makes me realize we can decouple the component from the state a bit and just get rid of the ID.